go/interpreter/lfvm/instructions_test.go

```diff
@@ -1261,35 +1261,38 @@ func TestInstructions_EIP2929_SSTOREReportsOutOfGas(t *testing.T) {
 	// Hence we cannot take the same testing approach as for the other operations in EIP-2929.
 
 	// SSTORE demands at least 2300 gas to be available
-	// 2301 gas is not enough to afford tosca.StorageDeleted.
-	for _, availableGas := range []tosca.Gas{2300, 2301} {
-		for revision := tosca.R09_Berlin; revision <= newestSupportedRevision; revision++ {
-			for _, access := range []tosca.AccessStatus{tosca.WarmAccess, tosca.ColdAccess} {
-				t.Run(fmt.Sprintf("%v/%v/%v", SSTORE, revision, access), func(t *testing.T) {
-
-					ctxt := context{
-						params: tosca.Parameters{
-							BlockParameters: tosca.BlockParameters{
-								Revision: tosca.R13_Cancun,
+	// 2301 gas is not enough to afford StorageAdded, StorageModified, StorageDeleted.
```

Comment:
This comment was difficult to read before, the new ordering is not much better:
shall we try something like this?
```go

tests := []tosca.Gas{
    2300, //< not enough for minimum gas requirement
    2301, //< not enough to afford StorageAdded, StorageModified, or StorageDeleted.
}
for _, availableGas := range tests{
```

Is the new for loop with storageStatus really needed?